packages/sdk/src/4.0/methodsInterface.ts

```diff
@@ -3458,12 +3461,12 @@ export interface ILooker40SDK extends IAPIMethods {
    *
    * GET /content_validation -> IContentValidation
    *
-   * @param fields Requested fields.
+   * @param request composed interface "IRequestContentValidation" for complex method parameters
    * @param options one-time API call overrides
    *
    */
   content_validation(
-    fields?: string,
+    request: IRequestContentValidation,
```

Comment:
This is a breaking SDK change

Reply:
sorry didn't see the top level comment.

How do you recommend we document?

Reply:
I can add it to the commit message so that it goes into the changelog.